app/routes/exercises.rb

```diff
@@ -59,6 +59,18 @@ class Exercises < Core
         redirect '/dashboard'
       end
 
+      post '/exercises/delete' do
+        exercises = UserExercise.find(params['exercise_ids'])
```

Comment:
Should this be deleting UserExercises or Exercises.  Based on the restful route looks like it should be deleting exercises

what do you think about scoping this to the current_user instead of doing the boolean check?
```
exercises = current_user.user_exercises.find(params['exercise_ids'])
```

Wat are you thoughts on calling these user_exercise_ids